stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -360,11 +383,12 @@ function externDecodePublicKey(KeyStore|TrustStore keyStore, handle keyAlias)
 } external;
 
 # Returns RSA encrypted value for the given data.
+# ```ballerina byte[]|crypto:Error cipherText = crypto:encryptRsaEcb(cipherText, publicKey); ```
 #
 # + input - The content to be encrypted
 # + key - Private or public key used for encryption
 # + padding - The padding
-# + return - Encrypted data or `Error` if key is invalid
+# + return - Encrypted data or `crypto:Error` if key is invalid
```

Comment:
```suggestion
# + return - Encrypted data or `crypto:Error` if key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.